e2e/commands/start/start_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package start_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ZupIT/horusec/internal/utils/testutil"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("running binary Horusec with start parameter", func() {
+	var (
+		session           *gexec.Session
+		err               error
+		flags             map[string]string
+		repoAuthorization string
+	)
+
+	JustBeforeEach(func() {
+		cmd := testutil.GinkgoGetHorusecCmdWithFlags(testutil.CmdStart, flags)
+		session, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+
+	})
+
+	When("--project-path is passed", func() {
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath: testutil.GoExample1,
+			}
+		})
+
+		It("Returns no error on execution", func() {
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Shows the project path property", func() {
+			Expect(session.Wait(2 * time.Minute).Out.Contents()).To(ContainSubstring("project_path"))
+		})
+
+		It("The path is set", func() {
+			Eventually(session).Should(gbytes.Say(fmt.Sprintf("%s", testutil.GoExample1)))
```

Comment:
This behavior can break the tests on Windows, because the path of Windows directories, has slashes "\" and at the junction of the characters, Ginkgo will understand it as a regex and not a string.

Example:

`D:\User\Lucas`
O ginko will interpret the `\U` as a regex.

Reply:
adjusted